pkgs/tools/misc/manga-cli/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenvNoCC
+, lib
+, fetchFromGitHub
+, makeWrapper
+, bash
+, img2pdf
+, zathura
+}:
+stdenvNoCC.mkDerivation {
+  pname = "manga-cli";
+
+  version = "1.0.5";
```

Comment:
```suggestion
  version = "1.0.5";
```

Reply:
Also, I think it would be better to call this version `unstable-YYYY-MM-DD` because the upstream doesn't set release tags. I understand that the version is [specified](https://github.com/7USTIN/manga-cli/blob/a69fe935341eaf96618a6b2064d4dcb36c8690b5/manga-cli#L3) in the script, but still. Maybe you can open an issue and ask the developer to set tags.